builder/googlecompute/builder_acc_test.go

```diff
@@ -284,3 +284,31 @@ func TestAccBuilder_WithMultipleDisks(t *testing.T) {
 	}
 	acctest.TestPlugin(t, testCase)
 }
+
+// Note: will be disabled unless an extra envvar is set to avoid timeouting
```

Comment:
I think this test could be added in a different PR, maybe also in a different file, since its not something we'd run in most instances, why do you want to add it in this PR?

Reply:
Oh yeah no that's a mistake on my part here, it should not have been part of this PR.
I'll remove it now.